pages/glossary/automatic-error-detection.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Automatic detection of input errors
+key: automatic-error-detection
+---
+
+Automatic detection of input errors is the result of automatically carried procedures to verify that the provided input meets the requirements. 
+
+The requirements include:
+
+- information that is required
+- information that must follow a specific format
+- information that must be above, below or within specific values
+- information that must belong to a set of allowed values
+
+The procedures can be:
+
+- Scripts executed in the client or the server;
+- Programmatically specified information that allows user agents to detect information that has not been provided or that must conform to a specific format (e.g. dates, phone numbers, etc.)
```

Comment:
This is kinda vague, can you clarify?

Reply:
I'm not sure what is vague in there (and I do have examples of how this can be done immediately after). Could you be more specific?